Userland/Libraries/LibWeb/WebGL/WebGLRenderingContextBase.cpp

```diff
@@ -20,4 +21,75 @@ WebGLRenderingContextBase::WebGLRenderingContextBase(HTML::HTMLCanvasElement& ca
 
 WebGLRenderingContextBase::~WebGLRenderingContextBase() = default;
 
+#define RETURN_IF_CONTEXT_LOST \
```

Comment:
this can be reduced to two lines when dropping the curly braces and doesn't seem worth a macro. What am I missing? :thinking: